dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/JsonTrackedEntityChangeLog.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2004-2022, University of Oslo
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ * Neither the name of the HISP project nor the names of its contributors may
+ * be used to endorse or promote products derived from this software without
+ * specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package org.hisp.dhis.webapi.controller.tracker;
+
+import org.hisp.dhis.jsontree.JsonObject;
+
+/** Representation of {@link org.hisp.dhis.tracker.export.event.EventChangeLog}. */
+public interface JsonTrackedEntityChangeLog extends JsonObject {
+  default JsonUser getCreatedBy() {
+    return get("createdBy").as(JsonUser.class);
+  }
+
+  default String getType() {
+    return getString("type").string();
+  }
+
+  default JsonChange getChange() {
+    return get("change").as(JsonChange.class);
+  }
+
+  interface JsonChange extends JsonObject {
+    default JsonTrackedEntityAttribute getTrackedEntityAttributeChange() {
+      return get("trackedEntityAttribute").as(JsonTrackedEntityAttribute.class);
```

Comment:
I would stick to our tracker naming in the API. https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/view/Attribute.java

we say `attribute` https://github.com/dhis2/dhis2-core/blob/3f52715324afbee980bc6843433a94984c5e022a/dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/view/TrackedEntity.java#L84

Reply:
don't pay too much attention to this test, it's still a work on progress, the current response looks like this:
![Screenshot from 2024-02-28 11-37-49](https://github.com/dhis2/dhis2-core/assets/2668498/25563d61-df34-4020-a2c5-e435686c7d4a), and it's already called attribute. I think that's the one you mean, right?